packages/app/src/components/profile/Profile.tsx

```diff
@@ -21,17 +27,34 @@ export const Profile = (props: Props): React.ReactElement => {
     );
 
     return (
-        <SafeAreaView>
-            <Text>{me.addresses[0]}</Text>
-            <Button
-                title="Start Streaming"
-                onPress={() =>
-                    props.navigation.navigate('CreateChallengeScreen', {
-                        me
-                    })
-                }
-            />
-            <UserChallengesList query={props.route.params.userChallengeQuery} />
-        </SafeAreaView>
+        <LinearGradient
```

Comment:
We might want a global background gradient that stays constant (similar to the other gradient background) . or have some kind of border around the page for swipe transitions. It looks funny right now

Reply:
Added a gradient behind maintabs but it looked the same, so I also added a fade to black between the Video screen and the other two. I think it will look less weird when the issue about the video being fixed during swipe is completed. Let me know what you think. Thanks!

Reply:
I should have been clearer which components. The swipe from profile to wallet looks funny

Reply:
Sounds good. Easy fix.

Reply:
Added the global gradient and it's been rebased with current master